amplify/backend/function/validateUserInvite/src/index.js

```diff
@@ -117,7 +97,31 @@ exports.handler = async (event) => {
   );
 
   if (contributorExistenceCheckQuery?.data?.getColonyContributor) {
-    // contributor already exists no need to create one
+    // If the user has already been a contributor for a Colony before,
+    // switching their isWatching field to true will allow them to access it again
+    const colonyMemberIsWatchingMutation = await graphqlRequest(
+      updateColonyContributor,
+      {
+        input: {
+          id: getColonyContributorId(colonyAddress, userAddress),
+          isWatching: true,
+        },
+      },
+      graphqlURL,
+      apiKey,
+    );
+
+    if (
+      colonyMemberIsWatchingMutation.errors ||
+      !colonyMemberIsWatchingMutation.data
+    ) {
+      const [error] = colonyMemberIsWatchingMutation.errors;
+
+      throw new Error(
+        error?.message || 'Could not set contributor isWatching field to true',
```

Comment:
So I don't think this is right. Since this is a graphql operation it should actually return `null` 

That's how graphql queries/mutation handle if the're an error processing them.

Also, since this is an lambda, you can safely log out errors to the console, as those will not be exposed in the client but rather in the AWS Cloudwatch logs, which helps us debug in production